### PR TITLE
Expand documentation

### PR DESCRIPTION
**Pull request by underwoo**
**Description**
Migration of the FMS description web page to a markdown file in the repository.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -14,16 +14,15 @@ The following external libraries are required when building libFMS
 
 We strive to have libFMS built with as many C and Fortran compilers on as many
 Unix/Linux type systems as possible.  However, internally, we only use libFMS
-compiled with the Intel compilers.  Some group have had success using libFMS
+compiled with the Intel compilers.  Some groups have had success using libFMS
 when compiled with the GNU C and Fortran compilers.
 
 ## MPI Support
 
 The default way to build libMPI is with MPI support.  We have found that using
```
Comment (bensonr) [inline INSTALL.md:22]: Should this be "libFMS with MPI support"?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,50 @@
...
+
+The Flexible Modeling System (FMS) is a software environment that supports the efficient development, construction, execution, and scientific interpretation of atmospheric, oceanic, and climate system models. This framework allows algorithms to be expressed on a variety of high-end computing architectures using common and easy-to-use expressions of the underlying platforms, spanning distributed and shared memory, as well as high-performance architectures. Scientific groups at GFDL can develop new physics and new algorithms concurrently, and coordinate periodically through this framework.
+
+![FMS Framework](docs/images/FMS.gif)
+
+Modeling frameworks for the construction of coupled models, made from independent model components, are now prevalent across this field. This framework, FMS, was one of the first to be developed — since the advent of the Cray T3E in 1998 — and is still in use and development today, using new architectures and new algorithms.
```
Comment (bensonr) [inline README.md:9]: I think it should be "in use and _under_ development"

File: INSTALL.md [hunk 3]
```diff
@@ -14,16 +14,15 @@ The following external libraries are required when building libFMS
...
 the MPI aware compiler, or the MPI compiler wrappers (mpif90, mpicc, etc.), in
 general, offer the best results when building with MPI support.  If you decide
-to not use an MPI aware you should pass the include and library locations to
-the `configure` script through the `CPPFLAGS`, `FCFLAGS` and `LDFLAGS` variables.
+to not use an MPI aware compiler, you should pass the include and library locations to the build system.
```
Comment (bensonr) [inline INSTALL.md:25]: May want to insert a hard return to limit the line length.

File: INSTALL.md (hunk 1 above)
Comment (bensonr) [reply INSTALL.md:22]: This is marked as resolved, but I still see "build libMPI with MPI support" in the text.

File: README.md [hunk 4]
```diff
@@ -1,10 +1,67 @@
-# Flexible Modeling System (FMS)
+# Modeling Framework: Flexible Modeling System (FMS)
+
+Today’s climate models simulate highly complex systems. In response to
+increasingly complex models, the climate community has developed tools and
+methodologies to facilitate the modeling process, developing modeling frameworks
```
Comment (wrongkindofdoctor) [inline README.md:5]: `, developing modeling frameworks
that provide infrastructure and tools to structure climate model codes and
facilitate many common tasks`--this sentence is redundant, and can be deleted

File: README.md [hunk 5]
```diff
@@ -1,10 +1,67 @@
...
+1998 — and is still in use and under development today, using new architectures
+and new algorithms.
+
+## What is FMS
 
 The Flexible Modeling System (FMS) is a software framework for supporting the
-efficient development, construction, execution, and scientific interpretation
-of atmospheric, oceanic, and climate system models.
+efficient development, construction, execution, and scientific interpretation of
+atmospheric, oceanic, and climate system models.  FMS comprises of the
```
Comment (wrongkindofdoctor) [inline README.md:32]: FMS ~~comprises~~ consists of

File: README.md [hunk 6]
```diff
@@ -1,10 +1,67 @@
...
+following:
+
+1. A software infrastructure for constructing and running atmospheric, oceanic,
```
Comment (wrongkindofdoctor) [inline README.md:35]: The numbering needs to be fixed

File: README.md [hunk 7]
```diff
@@ -1,10 +1,67 @@
...
+and climate system models. This infrastructure includes software to handle
+parallelization, input and output, data exchange between various model grids,
+orchestration of the time stepping, makefiles, and simple sample run scripts.
+This infrastructure should largely insulate FMS users from machine-specific
+details.
+1. A standardization of the interfaces between various component models.
```
Comment (wrongkindofdoctor) [inline README.md:41]: A standardization of the interfaces between various component models, including ~~S~~ software...

**Review by wrongkindofdoctor — CHANGES_REQUESTED** [review]

File: INSTALL.md (hunk 1 above)
Comment (underwoo) [reply INSTALL.md:22]: My bad.  I thought I fixed it with the other update.  I'll have it updated soon.

File: README.md (hunk 6 above)
Comment (bensonr) [reply README.md:34]: The numbering is fine when you view it out of the markup editor.

File: INSTALL.md (hunk 1 above)
Comment (underwoo) [reply INSTALL.md]: Update pushed.

File: README.md (hunk 6 above)
Comment (underwoo) [reply README.md:34]: Markdown will add the correct numbering when rendered.

**Comment by underwoo** [comment]
The changes have been made as requested.  Please review again.

I also added a `diag_table.md` file.  I pulled the information out of the diag_table.F90 file.

**Review by bensonr — APPROVED** [review]

**Review by wrongkindofdoctor — APPROVED** [review]

**Review by rem1776 — APPROVED** [review]

**Pull request merged**